sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeBatchActionsAsyncClient.java

```diff
@@ -375,6 +403,21 @@ private AnalyzeBatchActionsResult toAnalyzeTasks(AnalyzeJobState analyzeJobState
             }
         }
 
+        if (!CoreUtils.isNullOrEmpty(linkedEntityRecognitionTasksItems)) {
+            for (int i = 0; i < linkedEntityRecognitionTasksItems.size(); i++) {
+                final TasksStateTasksEntityLinkingTasksItem taskItem = linkedEntityRecognitionTasksItems.get(i);
+                final RecognizeLinkedEntitiesActionResult actionResult = new RecognizeLinkedEntitiesActionResult();
+                final EntityLinkingResult results = taskItem.getResults();
+                if (results != null) {
+                    RecognizeLinkedEntitiesActionResultPropertiesHelper.setResult(actionResult,
+                        toRecognizeLinkedEntitiesResultCollectionResponse(results));
```

Comment:
I'm lost... is `toRecognizeLinkedEntitiesResultCollectionResponse` the same method that is use when the sync `RecognizeLinkedEntities` method is called?

Reply:
You are right!  These two methods are most likely, I can merge them into one.